checker.c

```diff
@@ -1,21 +1,48 @@
 #include <stdio.h>
 #include <assert.h>
 
-int batteryIsOk(float temperature, float soc, float chargeRate) {
-  if(temperature < 0 || temperature > 45) {
-    printf("Temperature out of range!\n");
-    return 0;
-  } else if(soc < 20 || soc > 80) {
+void checkbattery_Temp(float temperature)
+{
+  if(temperature <0 || temperature > 45)
+  {
+    printf("Temperature out of range!\n"); 
+  }
+  else
+  {
+     printf("Temperature is in normal range!\n"); 
+  }
+}
+
+void checkbattery_soc(float soc)
+{
+  if(soc < 20 || soc > 80)
+  {
     printf("State of Charge out of range!\n");
-    return 0;
-  } else if(chargeRate > 0.8) {
+  }
+  else
+  {
+     printf("soc is in normal range!\n"); 
+  }
+}
+
+void batteryIsOk(float chargeRate) 
+{
+  if(chargeRate > 0.8) 
+  {
     printf("Charge Rate out of range!\n");
-    return 0;
   }
-  return 1;
+  else
+  {
+    printf("charge is in normal range!\n");
+  }
 }
 
-int main() {
-  assert(batteryIsOk(25, 70, 0.7));
-  assert(!batteryIsOk(50, 85, 0));
+
+
+int main() 
+{
+  checkbattery_Temp(25);
```

Comment:
There are no asserts here - that means we'll always need to prove the code by manually looking at the console output.
Assert the different conditions in the code